### DISCUSSION TITLE
search result suggestions cover example search patterns

### DISCUSSION THREAD

**Discussion by yaahc** [community/community#9744] [Other Feature Feedback, Questions, & Ideas]
Just started playing with code search, and I was trying to write out a more complicated search query but as soon as I started typing the suggestions drop-down completely covered the example I was using as a reference. Not a particularly big deal but it's definitely annoying to have to click on and off of the search text box so that I can keep reading the reference examples and continue filling out my search query.

![Screenshot from 2022-01-06 13-56-59](https://user-images.githubusercontent.com/1993852/148457852-0a0fbed3-95c8-4bc4-8ff5-41fa23d84c8c.png)

**Comment by kevinblount9** [comment]
That overlap issue feels like a small UX flaw, kind of like what [deepak shukla](https://deepakshukla.com/seo-consultant-london/) might flag during a technical SEO audit where visibility and accessibility are key to smooth navigation.

**Comment by AnshNikhare** [comment]
Hi @yaahc,

You’re right—having the suggestion dropdown cover the example search patterns can be frustrating when building complex queries. While this isn’t a blocking bug, it does impact usability.

**Quick tips:**
- If you need to keep referencing the examples, try opening the help/examples in a new tab or window as a temporary workaround.
- You can also use keyboard shortcuts like `Esc` to close the dropdown or click outside the box if you need to quickly reveal the covered examples.

**Feedback:**  
This is definitely a UI/UX detail worth improving. It might help to suggest making the dropdown reposition itself or adding an option to pin/unpin the examples for better visibility.

If you haven’t already, consider submitting this feedback via the “Give feedback” link on the code search page—direct input from users helps prioritize these fixes!

Hope this helps you work around the issue for now and gets it on the team’s radar for future updates!
